File: source/kCura.Relativity.Export/Exceptions/FileWriteError.vb
Namespace kCura.Relativity.Export.Exceptions
	Public Class FileWriteException
		Inherits ExportBaseException
		Public Enum DestinationFile
			Errors
			Load
			Image
			Generic
		End Enum
		Public Sub New(ByVal destination As DestinationFile, ByVal writeError As System.Exception)
			MyBase.new("Error writing to " & destination.ToString & " output file", writeError)
		End Sub

	End Class
End Namespace

File: source/kCura.Relativity.Export.ConvertedToC#/Exceptions/FileWriteError.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Exceptions
{
	public class FileWriteException : ExportBaseException
	{
		public enum DestinationFile
		{
			Errors,
			Load,
			Image,
			Generic
		}
		public FileWriteException(DestinationFile destination, System.Exception writeError) : base("Error writing to " + destination.ToString() + " output file", writeError)
		{
		}

	}
}
